.cirrus.yml

```diff
@@ -96,6 +96,7 @@ ci_template: &CI_TEMPLATE
 
   init_external_repos_script: ./ci/init-external-repos.sh
 
+  pre_build_script: ./ci/pre-build.sh
```

Comment:
Is this used by anything in 6.0? On master it's used in the `ubuntu22_spicy_head_task` task, but we don't have that task on 6.0 (or it wasn't added by this PR).

Reply:
Yeah you're right, let me drop it. Thanks!